### ISSUE TITLE
PlayPropAnimation callback crash

### ISSUE THREAD

**Issue by crater0** [jkulawik/tdd-issues#6]
PlayPropAnimation crashes when an animation over callback is set.

The following level script triggers the crash.

```
////////////////////////////
// Run when entering map
void OnStart()
{
	PlayPropAnimation("spider_1", "Idle", 1.0, false, "AnimCallback");
}

void AnimCallback(string &in asProp)
{
	AddDebugMessage("AnimCallback!", false);
}

////////////////////////////
// Run when leaving map
void OnLeave()
{
	
}
```

I also uploaded a sample map to the workshop that can be used to reproduce the crash.

https://steamcommunity.com/sharedfiles/filedetails/?id=3257341943

**Comment by jkulawik** [comment]
Luis has found a fix

**Comment by crater0** [comment]
Fixed in build-id:14564714
